test/test-files/address.schema.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
This and the other files inside of `test-files` are just snapshots for testing against.